lib/targetctl.js

```diff
@@ -210,15 +221,17 @@ function onRequest(req, callback) {
       // Unsupported
       default:
         rsp.error = 'unsupported';
+        callback(rsp);
         break;
     }
   } catch (err) {
     rsp.error = err.message;
+    callback(rsp);
+
   }
 
   debug('request %s => response %s', debug.json(req), debug.json(rsp));
 
-  callback(rsp);
```

Comment:
we had to change this due to the way appmetrics returned the cpu profiling as a callback.  The case 'stop-cpu-profiling' needs to wait for the callback before returning and so we had to move the callback(rsp) that was at the end of the code to be triggered in each case statement.  Otherwise, the 'stop-cpu-profiling' would return without waiting for the callback and an error would occur due to an empty profile